packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -214,6 +215,13 @@ export class EntityService {
     args: CreateOneEntityArgs,
     user: User
   ): Promise<Entity> {
+
+    if (args.data?.name?.toLowerCase() === args.data?.pluralDisplayName?.toLowerCase()) { 
```

Comment:
Since the value of plural display name may include spaces, you should remove white spaces before do the compare